### PR TITLE
Fix/login open browser

### PR DESCRIPTION
**Pull request by VerasThiago**
#### What is the purpose of this pull request?
Change `open` lib

#### What problem is this solving?
This library is deprecated and in some Linux distributions when I do the vtex login, it doesn't open the browser.

#### Types of changes
- [ ] Refactor (non-breaking change that only makes the code better)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

#### Chores checklist
- [X] Update `CHANGELOG.md`

### CODE REVIEW

File: src/nps.ts [hunk 1]
```diff
@@ -41,7 +43,15 @@ export async function checkAndOpenNPSLink() {
...
-      opn(NPSFormURL, { wait: false })
+      const configClient = ToolbeltConfig.createClient()
+      const { featureFlags } = await configClient.getGlobalConfig()
+
+      if (featureFlags.FEATURE_FLAG_NEW_OPEN_PACKAGE) {
+        open(NPSFormURL, { wait: false })
+      } else {
+        opn(NPSFormURL, { wait: false })
+      } 
+
```
Comment (dhasuda) [inline src/nps.ts]: _non blocking suggestion_:
The if with `featureFlags.FEATURE_FLAG_NEW_OPEN_PACKAGE` is used in a lot of places. You can make a selector somewhere else... in the future, if the logic gets more complex (perhaps you may need to use a _switch_ IDK)
```
const openPackageSelector(withFeatureFlag, withoutFeatureFlag) {
  // Logic to get feature flags...
 if (featureFlags.FEATURE_FLAG_NEW_OPEN_PACKAGE) {
    return withFeatureFlag
  } else {
    return withoutFeatureFlag
  }
}
```

**Review by dhasuda — APPROVED** [review]
LGTM! Only one non blocking coment

File: src/nps.ts (hunk 1 above)
Comment (VerasThiago) [reply src/nps.ts]: In my point of view, the use o `Feature Flags` in complex cases need to encapsulate the login in other functions, like [here](https://github.com/vtex/cli-plugin-deploy/blob/7e936891abd5a2cf0e77bd82216bc441a67cbf9a/src/commands/deploy.ts#L24), but in this case I don't think that needed. What do you think?

Comment (dhasuda) [reply src/nps.ts]: I don't think it's needed. But the suggestion is about creating a selector of the function that will be used. Something that analyses the feature flags and the possibilities of execution, and chooses one. The ideia is to simply separate the logic of the feature flag selection from the rest of the code

Comment (dhasuda) [reply src/nps.ts]: Anyway, it's not a blocking comment, feel free to move on

**Pull request merged**